<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-AACR0J1" name="test_cases.Test_New_Survey_Builder" tests="12" failures="1" timestamp="26 Feb 2016 21:43:09 GMT" time="31.833" errors="0">
  <testcase name="test_Select_EMA_App_Page" time="0.041" classname="test_cases.Test_New_Survey_Builder"/>
  <testcase name="test_Select_Survey_Builder" time="1.743" classname="test_cases.Test_New_Survey_Builder"/>
  <testcase name="test_Login" time="2.834" classname="test_cases.Test_New_Survey_Builder"/>
  <testcase name="test_Select_Card_Type" time="2.987" classname="test_cases.Test_New_Survey_Builder"/>
  <testcase name="test_Add_Card" time="1.296" classname="test_cases.Test_New_Survey_Builder">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at test_cases.Test_New_Survey_Builder.test_Add_Card(Test_New_Survey_Builder.java:184)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- test_Add_Card -->
  <testcase name="test_Create_New_EMA_Config" time="1.194" classname="test_cases.Test_New_Survey_Builder"/>
  <testcase name="test_Select_New_Project_Name" time="4.104" classname="test_cases.Test_New_Survey_Builder"/>
  <testcase name="test_Select_Instrument" time="0.924" classname="test_cases.Test_New_Survey_Builder"/>
  <testcase name="test_Add_New_Survey_Modal" time="7.145" classname="test_cases.Test_New_Survey_Builder"/>
  <testcase name="test_Select_EMA_Builder_Page" time="2.336" classname="test_cases.Test_New_Survey_Builder"/>
  <testcase name="test_Add_New_Section" time="6.575" classname="test_cases.Test_New_Survey_Builder"/>
  <testcase name="test_Select_New_EMA_Config" time="0.654" classname="test_cases.Test_New_Survey_Builder"/>
</testsuite> <!-- test_cases.Test_New_Survey_Builder -->
